app/services/idv/aamva_state_maintenance_window.rb

```diff
@@ -5,51 +5,57 @@ class AamvaStateMaintenanceWindow
     # All AAMVA maintenance windows are expressed in 'ET' (LG-14028),
     TZ = 'America/New_York'
 
+    Window = Data.define(:cron, :duration) do
+      def cron_val
+        Time.use_zone(TZ) { Fugit.parse_cron(cron) }
+      end
+    end.freeze
+
     MAINTENANCE_WINDOWS = {
       'AL' => [
         # First Monday of each month from 1 am – 7 am ET
-        { cron: '0 1 * * Mon#1', duration_minutes: 6 * 60 },
+        Window.new(cron: '0 1 * * Mon#1', duration: 6.hours),
       ],
       'CA' => [
         # Daily, 4:00 - 5:30 am. ET.
-        { cron: '0 4 * * *', duration_minutes: 90 },
+        Window.new(cron: '0 4 * * *', duration: 90.minutes),
         # Monday, 1:00 - 1:45 am. ET
-        { cron: '0 1 * * Mon', duration_minutes: 45 },
+        Window.new(cron: '0 1 * * Mon', duration: 45.minutes),
         # Monday, 1:00 - 4:30 am. ET on 1st and 3rd Monday of month.
-        { cron: '0 1 * * Mon#1', duration_minutes: 3.5 * 60 },
-        { cron: '0 1 * * Mon#3', duration_minutes: 3.5 * 60 },
+        Window.new(cron: '0 1 * * Mon#1', duration: 3.5.hours),
+        Window.new(cron: '0 1 * * Mon#3', duration: 3.5.hours),
       ],
       'CO' => [
         # 02:00 - 08:00 AM ET on the first Tuesday of every month.
-        { cron: '0 2 * * Tue#1', duration_minutes: 6 * 60 },
+        Window.new(cron: '0 2 * * Tue#1', duration: 6.hours),
       ],
       'CT' => [
         # Daily, 3:00 am. to 4:00 am. ET.
-        { cron: '0 3 * * *', duration_minutes: 1 },
+        Window.new(cron: '0 3 * * *', duration: 1.hour),
```

Comment:
An unexpected perk of moving to using a Duration is that doing the change semi-manually caused me to catch and fix this 1-minute vs. 1-hour mistake.